tools/ui-components/src/helpblock/helpblock.css

```diff
@@ -0,0 +1 @@
+@import '../colors.css';
```

Comment:
The colors are handled through tailwind, we don't need to import them separately.

Reply:
Since the helpBlock does not have styles that cannot be replicated using tailwind classes, having a cdd file is not necessary.